### PR TITLE
Fix update the model when only size of the sidebar is updated, 'Save'…

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
… button is not being enabled

FYI, bug found: https://github.com/dotCMS/core/issues/13944#issuecomment-386454457

### CODE REVIEW

File: src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts [hunk 1]
```diff
@@ -74,7 +76,7 @@ export class DotEditLayoutSidebarComponent implements OnInit, ControlValueAccess
      * @param {DotContainerColumnBox[]} containers
      * @memberof DotEditLayoutSidebarComponent
      */
-    updateContainers(containers: DotContainerColumnBox[]): void {
+    updateContainers(containers?: DotContainerColumnBox[]): void {
         this.propagateChange(this.getModel(containers));
     }
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts]: any test is updated with this change?

Comment (alfredo-dotcms) [reply src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts]: you're right, adding 2 more test for that!

File: src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts [hunk 2]
```diff
@@ -74,7 +76,7 @@ export class DotEditLayoutSidebarComponent implements OnInit, ControlValueAccess
      * @param {DotContainerColumnBox[]} containers
      * @memberof DotEditLayoutSidebarComponent
      */
-    updateContainers(containers: DotContainerColumnBox[]): void {
+    updateContainers(containers?: DotContainerColumnBox[]): void {
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts]: Does hhis method needs to be renamed and redocumented? @freddyucv @hmoreras let's talk about this.

File: src/app/portlets/dot-edit-page/layout/components/dot-sidebar-properties/dot-sidebar-properties.component.spec.ts [hunk 3]
```diff
@@ -65,9 +65,11 @@ describe('DotSidebarPropertiesComponent', () => {
     });
 
     it('should send a layout-sidebar-change notification when a sidebar size property is clicked', () => {
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/layout/components/dot-sidebar-properties/dot-sidebar-properties.component.spec.ts]: ```when a sidebar size property is clicked``` ❌ 

```when a sidebar size property is updated``` ✅

Comment (alfredo-dotcms) [reply src/app/portlets/dot-edit-page/layout/components/dot-sidebar-properties/dot-sidebar-properties.component.spec.ts]: done!

File: src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts (hunk 2 above)
Comment (alfredo-dotcms) [reply src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts]: mmm, this method was originally called liked that to propagate/update the containers to the parent, now it's been reused by 2 children to propagate any change than happened on the children components.

Maybe rename it to updateModel(). Thoughts?

**Pull request merged**